src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5454,6 +5454,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NullabilityMismatchInArgumentForOutput_Title" xml:space="preserve">
     <value>Argument cannot be used as an output for parameter due to differences in the nullability of reference types.</value>
   </data>
+  <data name="WRN_NullDisallowedInAssignment" xml:space="preserve">
```

Comment:
>WRN_NullDisallowedInAssignment [](start = 14, length = 30)

Consider using for specific name, reflecting that it is about DisallowNullAttribute.